crypto/bio/bss_log.c

```diff
@@ -390,11 +390,7 @@ static void xcloselog(BIO *bp)
 
 static void xopenlog(BIO *bp, char *name, int level)
 {
-#  ifdef WATT32                 /* djgpp/DOS */
     openlog(name, LOG_PID | LOG_CONS | LOG_NDELAY, level);
-#  else
-    openlog(name, LOG_PID | LOG_CONS, level);
-#  endif
```

Comment:
Doesn't this remove the wrong branch?

Reply:
I don't think so.  You sure>

Reply:
Not 100% sure since LOG_NDELAY is in posix and I don't know where it wouldn't be available. However, you keep the branch that was previously guarded by `WATT32` (another candidate for unifdef):
```diff
-#  ifdef WATT32                 /* djgpp/DOS */
     openlog(name, LOG_PID | LOG_CONS | LOG_NDELAY, level);
-#  else
-    openlog(name, LOG_PID | LOG_CONS, level);
-#  endif
```

Reply:
I guess I'm saying the pedantic way to deal with this would be to keep both branches and only drop the comment (or only `djgpp/`  if you prefer).

Reply:
You know how you can look at something forever and not see that it's wrong?  Sigh.  Thanks.  Pushed a commit that uses `ifdef LOG_NDELAY`